sw/device/silicon_creator/rom/keys/README.md

```diff
@@ -2,9 +2,35 @@
 title: "Signing Keys"
 ---
 
-This directory contains ASN1 DER encoded development keys for signing ROM\_EXT
-images. Until we have a more ergonomic tool for working with keys and signatures,
-below snippets can be used for various operations.
+# Generating keys with opentitantool
+
+[Open Titan Tool](ott) is the recommended tool to generate RSA keys to use in
+the `ROM`.
+
+Inspecting a key (show the public information of a key):
+```
+$ /path/to/opentitantool rsa key show <basename>.der
```

Comment:
Maybe we should use the bazel command
```
bazel run //sw/host/opentitantool -- rsa key show <basename>.der
```

Reply:
Yes I can change it, the only downside is that when running it, we need to specify `$PWD/<basename>.der`, I think because it is not executed in the current directory.

Reply:
fair enough